hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1108,6 +1108,64 @@ function upgrade_1010() {
       CRM_Hrjobcontract_JobContractDates::rewriteContactIds();
       return TRUE;
   }
+  
+  /**
+   * Adding 'end_reason' field for Job Contract Details,
+   * 'hrjc_contract_end_reason' Option Group with three Option Values 
+   * and Administration Menu entry for managing the Option Group values.
+   * 
+   * @return boolean TRUE
+   */
+  function upgrade_1011() {
```

Comment:
@coldrunKacper Please can we add a few comments to separate different sections in this function ? Thanks !!